contracts/common/FeePayoutSupport/FeePayoutSupportV1.sol

```diff
@@ -57,13 +62,19 @@ contract FeePayoutSupportV1 is OutgoingTransferSupportV1 {
     function _handleProtocolFeePayout(uint256 _amount, address _payoutCurrency) internal returns (uint256) {
         uint256 protocolFee = protocolFeeSettings.getFeeAmount(address(this), _amount);
 
-        if (protocolFee != 0) {
-            (, address feeRecipient) = protocolFeeSettings.moduleFeeSetting(address(this));
-            _handleOutgoingTransfer(feeRecipient, protocolFee, _payoutCurrency, 0);
+        // If fee switch off --
+        if (protocolFee == 0) {
```

Comment:
Inspired by [ESLint's `no-else-return` rule](https://eslint.org/docs/rules/no-else-return) and [Palantir's TSLint `unnecessary-else` rule](https://palantir.github.io/tslint/rules/unnecessary-else/) -- since the `if` block contains a `return` statement, the `else` block becomes unnecessary:

```suggestion
        // If fee switch off, return initial amount
        if (protocolFee == 0) {
            return _amount;
        }
        
        // Get fee recipient
        (, address feeRecipient) = protocolFeeSettings.moduleFeeSetting(address(this));
        // Payout protocol fee
        _handleOutgoingTransfer(feeRecipient, protocolFee, _payoutCurrency, 50000);
        // Return remaining amount
        return _amount - protocolFee;
```

Reply:
ty !